ui/src/components/MultiPanelTabs.vue

```diff
@@ -59,7 +72,7 @@
                 </button>
                 
                 <el-dropdown trigger="click" placement="bottom-end">
-                    <DotsVertical class="me-2 tab-icon" />
+                    <el-button :icon="DotsVertical" link class="me-2 tab-icon" />
                     <template #dropdown>
                         <el-dropdown-menu class="m-2">
                             <el-dropdown-item disabled :icon="DockRight">
```

Comment:
It would be great if you could extend your logic to make this button and the one below functional.

So, you'd need to remove disabled attribute from it, and to make the `Move right` button click move the pane one position to the right. Same thing for the left one.